docs/product/integrations/integration-platform/webhooks/issues.mdx

```diff
@@ -39,6 +39,23 @@ Sentry integrations that have subscribed to issue webhooks can receive informati
 
 ## Payload
 
+Payload includes additional information about the issue. Some important fields to pay attention to are:
+
+### status
+Some issue webhooks are triggered on status change of an issue. An issue status can be `resolved`, `unresolved`, or `ignored` (archived).
+
+### substatus
+A status can mean multiple things, so substatus gives more details about the status. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.
+
+### statusDetails
+`statusDetails` includes more detailed information about the action being done on the issue, for example is the issue is being resolved or archived. These details are:
+* `inRelease`: The release that the resolution is in for example `latest`
+* `inNextRelease`: `True` if the resolution will be in the next release
+* `inCommit`: Includes information about the resolution commit, for example `repository` and `commit`
+* `ignoreDuration`: Duration for an archived issue before it's turns to escalated in minutes
```

Comment:
```suggestion
* `inRelease`: The release that the resolution is in (for example, `latest`)
* `inNextRelease`: `True` if the resolution will be in the next release
* `inCommit`: Includes information about the resolution commit (such as `repository` and `commit`)
* `ignoreDuration`: The duration (in minutes) that an archived issue should be ignored before it gets escalated.
```

can you add more details about the behavior or ignoreDuration? For example, will a new occurrence still escalate the issue even if the duration hasn't been met? Or will a new occurrence trigger the duration countdown? If the issue is resolved during the ignoreduration, then what happens? Does it always escalate once the duration is up?